File: 卡码网/Python/test04.py
#4. A+B问题IV
import sys
from functools import reduce

def main():
	for line in sys.stdin:
		l=list(map(int,line.split()))
		if l[0]==0:
			break
		print(sum(l[1:]))

if __name__ == '__main__':
	main()

File: 卡码/KamaCoder/Js/test04.js
//4. A+B问题IV
const rl=require("readline").createInterface({input:process.stdin});
const iter=rl[Symbol.asyncIterator]();
const readline=async ()=>(await iter.next()).value;

async function main(){
  while(line=await readline()){
    const [n,...nums]=line.split(" ").map(Number);
    if(n===0) break;
    console.log(nums.reduce((item,prev)=>item+prev,0));
  }
}

main();